src/main/java/com/telefonica/iot/cygnus/backends/ckan/CKANCache.java

```diff
@@ -160,7 +168,9 @@ public boolean isCachedOrg(String orgName) throws Exception {
         
         // query CKAN for the organization information
         String ckanURL = "/api/3/action/organization_show?id=" + orgName;
-        CKANResponse res = requester.doCKANRequest("GET", ckanURL);
+        ArrayList<Header> headers = new ArrayList<Header>();
```

Comment:
Same as before :) There is quite some repeated code in this class too regarding composing the request, URL and querying CKAN which may recommend refactor it to some private methods so it can be reused... ;) As before, I leave it your discretion... ;)